config/default.yaml

```diff
@@ -24,7 +24,7 @@ services:
     clientID: 'idam_user_dashboard'
     clientSecret: 'idam_user_dashboard_secret'
     callbackURL: 'http://localhost:3100/oauth2/callback'
-    scope: 'openid profile roles manage-user search-user create-user'
+    scope: 'openid profile roles manage-user search-user create-user view-user'
```

Comment:
Might be easier to split the scope into `service_account_scopes` and `client_scopes` so you don't have to remove the `openid profile roles` scopes later on?

Reply:
I thought about that but I wanted to just get rid of the service account login eventually.
I can do it, I suppose, it just annoyed me to add more temporary code